frontend/src/pages/DCATWizard/Submission.jsx

```diff
@@ -0,0 +1,201 @@
+import React, { useState, useContext } from 'react';
+import styled from 'styled-components';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { history } from '../../router/history';
+import { alertActions } from '../../state/actions/alert';
+import { WizardContext } from './Context';
+import { FinalizedDataset } from './FinalizedDataset';
+import { LoadingButton } from '../../sharedComponents/LoadingButton';
+
+const Wrapper = styled.div`
+  height: 100%;
+  width: 100%;
+  padding: 10px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+`;
+
+const Datasets = styled.div`
+  padding: 5px;
+  overflow: auto;
+  flex: 1;
+`;
+
+const Submission = () => {
+  const reduxDispatch = useDispatch();
+  const userSelector = useSelector((state) => state.user);
+  const { token, municipalityName } = userSelector.user;
+
+  const { state } = useContext(WizardContext);
+
+  const [loading, setLoading] = useState(false);
+
+  // filter out the selected datasets
+  const selectedDatasets = state.datasetsState.datasets.filter((dataset) => (
+    state.datasetsState.selections.get(dataset.id)
+  ));
+
+  const onSubmit = async () => {
+    const catalogUUIDMapping = new Map();
+
+    setLoading(true);
+
+    try {
+      await Promise.all(state.mimetypes.map((mimetype) => fetch('/api/DataFormat', {
+        method: 'PUT',
+        mode: 'cors',
+        cache: 'no-cache',
+        credentials: 'same-origin',
+        headers: {
+          'Content-Type': 'application/json',
+          Authorization: `Bearer ${token}`,
+        },
+        redirect: 'follow',
+        referrerPolicy: 'no-referrer',
+        body: JSON.stringify({
+          name: mimetype.split('/')[1],
+          description: '[No Description]',
+          mimeType: mimetype,
+          documentationUrl: 'http://google.com/fuck+this',
```

Comment:
Probably better off left blank?
```suggestion
          documentationUrl: '',
```

Reply:
Can it be left blank? @petterroea

Reply:
It really shouldnt, but i guess we can change that.

Reply:
Or just change it to google.com, just don't think we should push needless profanity to our final product